packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -525,26 +525,38 @@ contract ChugSplashManager is Owned {
     }
 
     /**
-     * @notice Transfers ownership of a proxy from this contract to the project owner.
+     * @notice Transfers ownership of a proxy from this contract to an address selected by the
+     *         project owner.
      *
-     * @param _proxy     Proxy that is the subject of the ownership transfer.
-     * @param _proxyType The proxy's type.
-     * @param _newOwner  Address of the project owner that is receiving ownership of the proxy.
+     * @param _name     String name of the target that corresponds to the proxy.
+     * @param _newOwner Address of the project owner that is receiving ownership of the proxy.
      */
-    function transferProxyOwnership(
-        address payable _proxy,
-        bytes32 _proxyType,
-        address _newOwner
-    ) public onlyOwner {
+    function transferProxyOwnership(string memory _name, address _newOwner) public onlyOwner {
```

Comment:
The main change is that this function now takes in a target's string name as input instead of the proxy and proxyType.  I think this makes things more consistent with other parts of the contract that do the same.